### PR TITLE
pkg/operators: Add tests for SortOperators

### PR DESCRIPTION
**Pull request by burak-ok**
Tests the sorting of Operators

### CODE REVIEW

File: pkg/operators/operators_test.go [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+		createOp("a", []string{}),
+	}
+
+	sortedOps, err := SortOperators(ops)
+	if assert.NoError(t, err) {
+		checkDependencies(t, ops, sortedOps)
+	}
+}
+
+func Test_SortOperatorsLargaGraph(t *testing.T) {
```
Comment (flyth) [inline pkg/operators/operators_test.go:144]: nit:
```suggestion
func Test_SortOperatorsLargeGraph(t *testing.T) {
```

File: pkg/operators/operators_test.go [hunk 2]
```diff
@@ -0,0 +1,195 @@
+// Copyright 2022 The Inspektor Gadget authors
```
Comment (flyth) [inline pkg/operators/operators_test.go:1]: nit:
```suggestion
// Copyright 2022-2023 The Inspektor Gadget authors
```

Sorry that it took so long :)

**Review by flyth — APPROVED** [review]
Thanks! This is also a good base for other future tests concerning operators.

I like that you're also using the testify lib already that we just discussed in #1355.

File: pkg/operators/operators_test.go (hunk 2 above)
Comment (burak-ok) [reply pkg/operators/operators_test.go:1]: Haha, typical copy paste error :D

**Pull request merged**